Sharpmake.Generators/FastBuild/Bff.cs

```diff
@@ -1678,7 +1678,9 @@ Strings ignoreSpecificLibraryNames
                     // here we could also verify that the path is rooted
                     if (extension != platformVcxproj.StaticLibraryFileFullExtension && extension != platformVcxproj.SharedLibraryFileFullExtension)
                     {
-                        libraryFile = libPrefix + libraryFile;
+                        string dir = Path.GetDirectoryName(libraryFile);
+                        string file = Path.GetFileName(libraryFile);
```

Comment:
I wonder if that's the right thing to do, maybe we should throw an error in case the path is rooted? (as said line 1678)